### PR TITLE
Add `robot` icon for Service Principals

### PR DESCRIPTION
**Pull request by jorytindall**
### :pushpin: Summary

If merged this PR adds a robot icon to be used for Service Principals

### :camera_flash: Screenshots

Screenshot 2023-06-16 at 4 48 05 PM

### :link: External links

Jira ticket: [HDS-1898](https://hashicorp.atlassian.net/browse/HDS-1898)
Figma file: [Flight icons](https://www.figma.com/file/TLnoT5AYQfy3tZ0H68BgOr/Flight-Icons?type=design&node-id=350%3A2&t=8eY21iSHlbHDJwZX-1)

***

### 👀 Reviewer's checklist:

- [ ] +1 Percy if applicable
- [ ] Confirm that PR has a changelog update via [Changesets](https://github.com/changesets/changesets) if needed

:speech_balloon: Please consider using [conventional comments](https://conventionalcomments.org/) when reviewing this PR.

[HDS-1898]: https://hashicorp.atlassian.net/browse/HDS-1898?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by didoo — APPROVED** [review]
❤️

**Review by cveigt — APPROVED** [review]
🤖

File: packages/flight-icons/.env.template [hunk 1]
```diff
@@ -1,7 +0,0 @@
-# This is an api key created from the account of [your name here]
```
Comment (Dhaulagiri) [inline packages/flight-icons/.env.template:1]: @jorytindall this file should not be removed

Comment (jorytindall) [reply packages/flight-icons/.env.template:1]: Thanks for catching that, I failed to duplicate it and just renamed it.

**Pull request merged**